### PR TITLE
[skip ci] improve milvus.proto comments

### LINKED ISSUE #7449: [DOC] Add a documentation to explain how Milvus handles the proto
#### What would you like to be added:

I want to add a documentation to explain how Milvus handles the proto.

#### Why is this needed:

This will tell users and develpers the data transfer protocol in Milvus. Especially for sdk develpers, this documentation will guide them how to pass the parameter more clear.

### PR DESCRIPTION
**Pull request by nameczz**
Signed-off-by: nameczz <zizhao.chen@zilliz.com>
Resolves: #7449

### CODE REVIEW

File: internal/proto/milvus.proto [hunk 1]
```diff
@@ -141,26 +141,49 @@ message DescribeCollectionResponse {
   int32 shards_num = 8; 
 }
 
+/**
+* Load collection data into cache, then you can vector search by this collection.
```
Comment (DragonDriver) [inline internal/proto/milvus.proto:145]: "then you can do vector search on this collection" may be better.

Comment (DragonDriver) [inline internal/proto/milvus.proto:145]: "Load collection data into query nodes" may be better.

Comment (DragonDriver) [reply internal/proto/milvus.proto:145]: And we can mention that a collection cannot be searched if this collection was not loaded.

File: internal/proto/milvus.proto [hunk 2]
```diff
@@ -141,26 +141,49 @@ message DescribeCollectionResponse {
...
+  common.MsgBase base = 1;
+  // Not useful for now
   string db_name = 2;
-  string collection_name = 3; // must
+  // The collection name you want to load
+  string collection_name = 3;
 }
 
+/**
+* Release collection data from cache, then you can not vector search by this collection.
```
Comment (DragonDriver) [inline internal/proto/milvus.proto:157]: ditto.

File: internal/proto/milvus.proto (hunk 1 above)
Comment (nameczz) [reply internal/proto/milvus.proto]: > And we can mention that a collection cannot be searched if this collection was not loaded.

I think user can realize that.

**Comment by yanliang567** [comment]
manually label ci-passed for only comments updated

**Pull request merged**